var/spack/repos/builtin/packages/py-setuptools-rust/package.py

```diff
@@ -12,6 +12,8 @@ class PySetuptoolsRust(PythonPackage):
     homepage = "https://github.com/PyO3/setuptools-rust"
     pypi = "setuptools-rust/setuptools-rust-0.12.1.tar.gz"
 
+    version("1.4.1", sha256="18ff850831f58ee21d5783825c99fad632da21e47645e9427fd7dec048029e76")
+    version("1.2.0", sha256="a47d72d3723d03c7f66c283e301bbbf4260c2a0e7f02cda30ac2b8f65e42720b")
```

Comment:
This sha256 is incorrect.

```suggestion
    version("1.2.0", sha256="0a4ada479e8c7e3d8bd7cb56e1a29acc2b2bb98c2325051b0cdcb57d7f056de8")
```